### ISSUE TITLE
Login to CM2000 - Failing :: loginUrl = loginForm[0].get("action")

### ISSUE THREAD

**Issue by jconsolatti** [andresp/cablemodem-status#9]
Doesn't look like the login function can reliability pull the cookie from the CM200. 

INFO:app:Logging into modem
[] <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<< Added :: print(loginPage.select("#target"))
Traceback (most recent call last):
  File "/Volumes/WinBucket/files/script-dev/CableModem/retriever.py", line 22, in wrapper
    return job_func(*args, **kwargs)
  File "/Volumes/WinBucket/files/script-dev/CableModem/retriever.py", line 42, in collectionJob
    modem.login()
  File "/Volumes/WinBucket/files/script-dev/CableModem/modems/netgear_cm2000.py", line 148, in login
    loginUrl = loginForm[0].get("action")
IndexError: list index out of range

~~
Tried to loop through the login process until it is != [] .. Need to debug some more to see if I can figure out what is going on .. 

while loginPage.select("#target") == []:
            # response = []
            time.sleep(10)
            response = self.session.get(self.baseUrl, verify=False)
            loginPage = BeautifulSoup(response.content, features="lxml")
            print(loginPage.select("#target"))

~~

--|--
Cable Firmware Version | V8.01.02
-- | --

**Comment by jconsolatti** [comment]
Might me a timing issue while testing in somewhat rapid succession while testing data collection on my side .. 

I am catching the  loginPage.select("#target") == []: and triggering a second auth attempt that gets me the cookie .. 

    def login(self):
        self.logger.info("Logging into modem")

        modemAuthentication = {
            'loginName': "admin",
            'loginPassword': self.config['Modem']['Password']
        }

        response = self.session.get(self.baseUrl, verify=False)

        # Get login url
        loginPage = BeautifulSoup(response.content, features="lxml")

        print("Before Loop: ", loginPage.select("#target"))

        while loginPage.select("#target") == []:
            response = []
            time.sleep(2)
            response = self.session.get(self.baseUrl, verify=False)
            loginPage = BeautifulSoup(response.content, features="lxml")
            print(loginPage.select("#target"))

        loginForm = loginPage.select("#target")
        loginUrl = loginForm[0].get("action")

        response = self.session.post(self.baseUrl + loginUrl, data=modemAuthentication, verify=False)

**Comment by jhgrazier** [comment]
I have been seeing the error inconsistently and a different error.

```
Traceback (most recent call last):
  File "/root/cablemodem-status/retriever.py", line 22, in wrapper
    return job_func(*args, **kwargs)
  File "/root/cablemodem-status/retriever.py", line 42, in collectionJob
    modem.login()
  File "/root/cablemodem-status/modems/netgear_cm2000.py", line 136, in login
    loginUrl = loginForm[0].get("action")
IndexError: list index out of range

INFO:app:Running as daemon
^CTraceback (most recent call last):
  File "/root/cablemodem-status/retriever.py", line 112, in <module>
    time.sleep(1)
KeyboardInterrupt

# python3 retriever.py
INFO:app:Reading configuration
INFO:app:Connecting to InfluxDB
INFO:app:Connecting to Selenium remote
INFO:app:Logging into modem
INFO:app:Getting modem status
Traceback (most recent call last):
  File "/root/cablemodem-status/retriever.py", line 22, in wrapper
    return job_func(*args, **kwargs)
  File "/root/cablemodem-status/retriever.py", line 44, in collectionJob
    modem.collectStatus()
  File "/root/cablemodem-status/modems/netgear_cm2000.py", line 163, in collectStatus
    upstreamQamPoints = self.formatUpstreamQamPoints(upstreamQamChannels, sampleTime)
  File "/root/cablemodem-status/modems/netgear_cm2000.py", line 54, in formatUpstreamQamPoints
    .tag("symbolRate", int(row[4])) \
ValueError: invalid literal for int() with base 10: '2560 Ksym/sec'

```

**Comment by andyp-79** [comment]
The parsing issue should be fixed in the latest release now: 0.0.21.
